src/libcore/slice.rs

```diff
@@ -655,6 +655,12 @@ impl<'a, T> Default for &'a [T] {
     fn default() -> &'a [T] { &[] }
 }
 
+#[stable(feature = "rust1", since = "1.5.0")]
+impl<'a, T> Default for &'a mut [T] {
+    #[stable(feature = "rust1", since = "1.5.0")]
```

Comment:
Yeah the methods on a trait impl don't need these annotations, and can you also rename the "rust1" feature to something like "mut_slice_default"? Currently we require that a feature name is all stabilized within the same "since".